### ISSUE TITLE
Popover onblur not working in chrome

### ISSUE THREAD

**Issue by nawazishalam** [reactstrap/reactstrap#1671]
component: Popover
react version : 16.7.0
reactstrap version:  7.1.0

Popover not closed on blur. Please need help to make this fix.

I need to close Popover closed on blur. 

Thanks

**Comment by nawazishalam** [comment]
`trigger="focus"` is basically a UncontrolledPopover and it does the expected behavior. Unfortunately I must to use _controlled popover_ and achieve the expected behavior. 

I used simple example of controlled popover [http://reactstrap.github.io/components/popovers/#Popover](url)

**Comment by TheSharpieOne** [comment]
Yes, the example use `UncontrolledPopover` for those specific examples, but the trigger also works with `Popover` itself.
Here is the simple example from controlled popover but with the `trigger="focus"` prop:
https://stackblitz.com/edit/reactstrap-v8-crepjx?file=Example.js

`trigger` is used to determine when the `toggle` callback is called. On a regular `Popover` the control/state is still 100% in your component.

**Comment by thefilmmaking** [comment]
Was having a similar issue, but `trigger="legacy"` fixed the issue. Somehow my controlled popover was having trouble with toggling `isOpen` state with the `trigger="focus"` (I have a peculiar use case), but the `legacy` trigger resolved all issues for me.
